Library/Homebrew/cask/test/support/Casks/with-depends-on-arch-failure.rb

```diff
@@ -1,12 +0,0 @@
-test_cask 'with-depends-on-arch-failure' do
-  version '1.2.3'
-  sha256 '67cdb8a02803ef37fdbf7e0be205863172e41a561ca446cd84f0d7ab35a99d94'
-
-  url "file://#{TEST_FIXTURE_DIR}/cask/caffeine.zip"
-  homepage 'http://example.com/with-depends-on-arch-failure'
-
-  # guarantee mismatched hardware
```

Comment:
I’ve removed this test because this is no longer possible. If there is only one option, we can’t have multiple valid options that are mutually exclusive.

There is [a different test for invalid values](https://github.com/Homebrew/brew/blob/master/Library/Homebrew/cask/test/support/Casks/invalid/invalid-depends-on-arch-value.rb).